app/src/main/java/com/naman14/hacktoberfest/fragment/ExploreFragment.java

```diff
@@ -151,6 +154,44 @@ public boolean onSelection(MaterialDialog dialog, View view, int which, CharSequ
                 .show();
     }
 
+    @OnClick(R.id.tv_tags)
+    public void showTagsDialog() {
+        final List<String> tagsList = Arrays.asList(Utils.getTagsArray());
+        String[] tagsPreference = Utils.getTagsPreference(getActivity());
+        Integer[] tagsPreferenceIndex = {};
+
+        if (tagsPreference != null) {
+            tagsPreferenceIndex = new Integer[tagsPreference.length];
+
+            int index = 0;
+            for (String tagPreference: tagsPreference) {
+                tagsPreferenceIndex[index++] = tagsList.indexOf(tagPreference);
+            }
+        }
+
+        new MaterialDialog.Builder(getActivity())
+                .title("Select Tags")
+                .items(Utils.getTagsArray())
+                .itemsCallbackMultiChoice(tagsPreferenceIndex
```

Comment:
weird formatting

Reply:
Fixed in 58883bc657b707111fa007deb1b5ad26e180dd6a